tests/unit/test-debug-info-test.js

```diff
@@ -0,0 +1,208 @@
+import { module, test } from 'qunit';
+import { run } from '@ember/runloop';
+import DebugInfo, { getDebugInfo } from '@ember/test-helpers/-internal/debug-info';
+import MockStableError, { overrideError, resetError } from './utils/mock-stable-error';
+import { MockConsole, getRandomBoolean, getMockDebugInfo } from './utils/test-isolation-helpers';
+
+module('TestDebugInfo', function() {
+  test('summary returns minimal information when debugInfo is not present', function(assert) {
```

Comment:
This module should have a beforeEach that sets run.backburner.DEBUG = false (since the tests rely on it being off).

Reply:
Done. Also added `run.backburner.DEBUG = true` for the nested module's `beforeEach`, since it relies on it.